### PR TITLE
[AAP-11836] Add hasCheckIcon to the Page Action Switch

### PR DESCRIPTION
**Pull request by lgalis**
Add hasCheckIcon to the Page Action Switch

Screenshot from 2023-08-07 18-02-39

### CODE REVIEW

**Comment by jamestalton** [comment]
I'm checking with UXD if we might just want to turn in on for all switches. The extra flag might make it so that devs forget that flag and then the switches across the UIs would look different.

**Review by tiyiprh — APPROVED** [review]
LGTM!

File: framework/PageActions/PageActionSwitch.tsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ export function PageActionSwitch<T extends object>(props: {
             label={label}
             labelOff={labelOff}
             isChecked={isChecked}
+            hasCheckIcon={true}
```
Comment (vidyanambiar) [inline framework/PageActions/PageActionSwitch.tsx:45]: Is this needed? I already see the check icon on action of type PageActionType.Switch

Comment (lgalis) [reply framework/PageActions/PageActionSwitch.tsx:45]: @vidyanambiar - the flag is necessary for the top of the page action swicth - as seen in the screenshot included in the PR's description.

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**